JSONObject.java

```diff
@@ -276,16 +277,20 @@ public JSONObject(Map<?, ?> m) {
      * <code>"is"</code> followed by an uppercase letter, the method is invoked,
      * and a key and the value returned from the getter method are put into the
      * new JSONObject.
-     *
+     * <p>
      * The key is formed by removing the <code>"get"</code> or <code>"is"</code>
      * prefix. If the second remaining character is not upper case, then the
      * first character is converted to lower case.
-     *
+     * </p><p>
      * For example, if an object has a method named <code>"getName"</code>, and
      * if the result of calling <code>object.getName()</code> is
      * <code>"Larry Fine"</code>, then the JSONObject will contain
      * <code>"name": "Larry Fine"</code>.
-     *
+     * </p><p>
+     * Methods that return <code>void</code> as well as <code>static</code>
+     * methods are ignored.
+     * </p>
```

Comment:
ah, I see the javadoc command in java9 will instert the close tags for you. I'll get rid of them